client/src/index.tsx

```diff
@@ -1,17 +1,36 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
 import * as serviceWorker from './serviceWorker';
+import App from './App';
+import { Drizzle } from '@drizzle/store';
+// import MyStringStore from './contracts/MyStringStore.json';
```

Comment:
import of json didnt work, switch to require and it seems to work???? just guessing

Reply:
yeah I would recommend having a `adapter` file, eg:

```ts
// client/src/json/index.js
const MyStringStore = require('../contracts/MyStringStore.json');

export { MyStringStore };
```

then in this file you can do:

```tsx
import { MyStringStore } from './json';
```